tests/pytests/test_coordinator.py

```diff
@@ -57,3 +57,11 @@ def testCommandStatsOnRedis(env):
 
     conn.execute_command('FT.INFO', 'idx')
     check_info_commandstats(env, 'FT.INFO')
+
+def test_curly_brackets(env):
+    conn = getConnectionByEnv(env)
+    env.expect('FT.CREATE', 'idx', 'SCHEMA', 't', 'TEXT', 'SORTABLE').ok()
+
```

Comment:
Should we also test the "breaking behavior"?
```suggestion
conn.execute_command('FT.ADD', 'idx', 'foo{baz}', '1.0', 'FIELDS', 't', 'Ciao mondo!')
env.expect('FT.SEARCH', 'idx', 'mondo').equal([1, 'foo{baz}', ['t', 'Ciao mondo!']])
```

Reply:
As discussed, `FT.ADD` isn't supported anymore and is using `HSET` in the background so no need for an additional test.